components/chart/src/chart.ts

```diff
@@ -41,25 +41,31 @@ import {
   ViewChild,
   ViewEncapsulation,
 } from '@angular/core';
+import {
+  createInViewportStream,
+  DtViewportResizer,
+  isDefined,
+} from '@dynatrace/barista-components/core';
+import { DtTheme } from '@dynatrace/barista-components/theming';
 // tslint:disable-next-line:no-duplicate-imports
 import * as Highcharts from 'highcharts';
 // tslint:disable-next-line:no-duplicate-imports
 import {
-  ChartObject,
-  Options as HighchartsOptions,
-  IndividualSeriesOptions,
   addEvent as addHighchartsEvent,
   chart,
+  ChartObject,
```

Comment:
There is no import sorter, but due to the fact that my build breaks on every unused import, i use order imports of vs code to let my build pass. @ffriedl89, @thomaspink in my opinion, we should discuss this in a separate topic in case that this slows down developer productivity extremely!